andela/-Healthchecks_BauMeister-
# #141392723 Adding email sending functionality with Sendgrid

#### What does this PR do?

Adds email sending functionality with Sendgrid.

#### Description of Task to be completed?

The application should be able to send emails using SendGrid API

#### How should this be manually tested?

- Visit the project's heroku web address:

    [healthchecks-bau.herokuapp.com](http://healthchecks-bau.herokuapp.com/)

- Click the `Log In` button.

- Enter a valid e-mail address.

- Click the `Log In` button.

- Visit your email for the log in instructions. *(Check in the junk/spam folder just in case)*

- Click the `Log In` button in the received email.

- You will be re-directed to the project's heroku web address and logged in automatically.
